packages/ibm-products/src/components/RemoveModal/RemoveModal.tsx

```diff
@@ -211,6 +218,7 @@ export let RemoveModal = forwardRef(
           <Button
             type="submit"
             kind="danger"
+            dangerDescription={primaryDangerDescription}
```

Comment:
```suggestion
            dangerDescription={primaryDangerDescription || undefined}
```
or undefined, so that the default `danger` will be rendered, if the prop is not provided or is empty.

Reply:
right!! my storybook was having this prop persistent in the url that it was setting it to empty string while testing it.